### PR TITLE
[SHRINKRES-317] Find a way to update all poms in all subdirectories

### PR DESCRIPTION
**Pull request by petrberan**
https://issues.redhat.com/browse/SHRINKRES-317

Opening as a draft for discussion @xstefank

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -385,6 +406,32 @@
...
+        <profile>
+            <id>unix</id>
+            <activation>
+                <os>
+                    <family>unix</family>
+                </os>
+            </activation>
+            <properties>
+                <!-- Set the properties for Unix-like systems (including Linux and macOS) -->
+                <os.executable>bash</os.executable>
```
Comment (xstefank) [inline pom.xml:431]: sh?

**Review by xstefank — APPROVED** [review]
I would remove bat since you don't have use case for it. Or if you can prove that it works than you can keep it.

**Comment by petrberan** [comment]
Removed the .bat

File: pom.xml (hunk 1 above)
Comment (xstefank) [reply pom.xml]: not particularly but more portable

**Review by xstefank — APPROVED** [review]
Ok, let's give this a chance. It would still be good to see if we can somehow automate it.

**Comment by xstefank** [comment]
@petrberan I can't merge draft PRs.

File: pom.xml (hunk 1 above)
Comment (petrberan) [reply pom.xml]: Fixed

**Pull request merged**